gemfiles/activerecord-5.0/Gemfile.base

```diff
@@ -1,3 +1,3 @@
 eval File.read File.expand_path('../../Gemfile.base', __FILE__)
 
-gem "activerecord", "~> 4.2.6"
+gem "activerecord", ">= 5.0.0.beta1", "< 5.1"
```

Comment:
@ronen I think this is an issue with ` schema_dev`. This project's `Gemfile` specifies `~> 5.0`.